Packs/MicrosoftGraphMail/Integrations/MicrosoftGraphMail/MicrosoftGraphMail.yml

```
@@ -9,17 +9,18 @@ configuration:
   required: true
   type: 0
 - display: ID or Client ID - see Detailed Instructions (?)
-  name: auth_id
-  required: true
-  type: 4
+  name: _auth_id
+  required: false
+  type: 0
 - display: Token or Tenant ID - see Detailed Instructions (?)
-  name: tenant_id
-  required: true
-  type: 4
-- display: Key or Client Secret - see Detailed Instructions (?)
-  name: enc_key
-  required: true
-  type: 4
+  name: _tenant_id
+  required: false
+  type: 0
+- displaypassword: Key or Client Secret - see Detailed Instructions (?) (Recommended)
```

```suggestion
- displaypassword: Key or Client Secret - see Detailed Instructions (?)
```
if the old one is hidden, no need to specify this